magentoerpconnect/product.py

```diff
@@ -631,6 +631,16 @@ def _must_skip(self):
 
         :returns: None | str | unicode
         """
+        prefix = self.env['ir.config_parameter'].get_param(
+            'outlet_product_prefix')
+        if prefix and self.magento_record['sku'] and \
+                self.magento_record['sku'].startswith(prefix + '-'):
```

Comment:
I think there might be a chance that the key is not in the dictionary at all, so it's safer to use `self.magento_record.get('sku')`

Reply:
thx